tests/snapshots/run_file__nested_str.hvm.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_file/nested_str.hvm
+---
+Invalid readback: [InvalidStrTerm]
```

Comment:
Shouldn't we output malformed strings in their constructor form? Since we can expect them to happen for example in partially applied functions.

Reply:
Just fixed this, will add a commit soon